alpenhorn/auto_import.py

```diff
@@ -470,6 +533,22 @@ def _import_file(node, root, acq_name, file_name):
                     "empty. Leaving fields NULL." % (acq_name)
                 )
                 di.CorrAcqInfo.create(acq=acq)
+    if atype == "hfb" and ftype.name == "hfb":
```

Comment:
```suggestion
    elif atype == "hfb" and ftype.name == "hfb":
```
This is part of one big if-elif statement running though all the acq types